pom.xml

```diff
@@ -80,6 +80,12 @@
       </dependency>
       <dependency>
         <groupId>javax.inject</groupId>
+        <artifactId>javax.inject</artifactId>
+        <classifier>sources</classifier>
```

Comment:
this is already in the `pom.xml` above. Adding it gives us both `javax.inject:javax.inject:1` and `javax.inject:javax.inject:sources:1`. Shouldn't this just be inlined in the gwt/pom.xml? It doesn't seem fitting for `<dependencyManagement>`. Especially since we hope to move off Maven soon anyway I think it will be more clear to just inline it.